build-aux/snap/snapcraft.yaml

```diff
@@ -1,21 +1,20 @@
 name: snapd
+# TODO: enable this when the store is ready
+# type: snapd
 summary: Daemon and tooling that enable snap packages
 description: |
- Install, configure, refresh and remove snap packages. Snaps are
- 'universal' packages that work across many different Linux systems,
- enabling secure distribution of the latest apps and utilities for
- cloud, servers, desktops and the internet of things.
- 
- Start with 'snap list' to see installed snaps.
-version: set-by-version-script-thxbye
-version-script: |
-  ./mkversion.sh --output-only
-#FIXME: enable once snapcraft understands this
-#type: snapd
-grade: stable
+  Install, configure, refresh and remove snap packages. Snaps are
+  'universal' packages that work across many different Linux systems,
+  enabling secure distribution of the latest apps and utilities for
+  cloud, servers, desktops and the internet of things.
 
-passthrough:
-  license: GPL-3.0
+  Start with 'snap list' to see installed snaps.
+adopt-info: snapd
+# the base is only needed here for snapcraft to check the ELF sections of
+# binaries inside the snap, which is an unnecessary check for snapcraft itself
+base: core
```

Comment:
I think this should be `build-base: core`, no?

Reply:
With just `build-base: core` snapcraft errors like this https://pastebin.ubuntu.com/p/PhMkTmKcRw/

I believe this is a bug in snapcraft where it assumes that if there is a build-base defined then there must also be a base defined. Should the snapd snap not declare a base?

Reply:
No, this was n'acked in my original PR, see https://github.com/snapcore/snapd/pull/7092#issuecomment-516735977